### PR TITLE
fix set_theia_version script, set inversify version to '4.14.0'

### PR DESCRIPTION
**Pull request by evidolob**
This PR is part of preparation to have proper release
Signed-off-by: Yevhen Vydolob <yvydolob@redhat.com>

### CODE REVIEW

File: set_theia_version.sh [hunk 1]
```diff
@@ -77,28 +77,38 @@ change_change_che_theia_init_baranch () {
...
-sed -i -e "s/ARG THEIA_VERSION=..*/ARG THEIA_VERSION=${theiaVersion}" ./dockerfiles/theia/Dockerfile
+# sed -i -e "s/RUN git clone -b 'master'/RUN git clone -b 'v${theiaVersion}'/" ./dockerfiles/theia-dev/e2e/Dockerfile
+# sed -i -e "s/ARG THEIA_VERSION=..*/ARG THEIA_VERSION=${theiaVersion}/" ./dockerfiles/theia/Dockerfile
 
-for dir in extensions/*
-do
-    echo ${dir}
-    updateTheiaDependencies ${dir} ${theiaVersion}
-done
+# for dir in extensions/*
```
Comment (benoitf) [inline set_theia_version.sh]: if commented, just delete it no ?
it's still in git history

File: plugins/task-plugin/package.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
     "backend": "lib/task-plugin-backend.js"
   },
   "dependencies": {
-    "inversify": "4.13.0",
+    "inversify": "4.14.0",
```
Comment (RomanNikitenko) [inline plugins/task-plugin/package.json]: We have approved CQ for 4.13.0 https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19424
Should we open CQ for 4.14.0?

File: set_theia_version.sh (hunk 1 above)
Comment (evidolob) [reply set_theia_version.sh]: Oh, no, I forgot to uncomment that code

File: plugins/task-plugin/package.json (hunk 2 above)
Comment (evidolob) [reply plugins/task-plugin/package.json]: I think yes, but we need TypeA CQ check, I change that version to be aligned with Theia

Comment (evidolob) [reply plugins/task-plugin/package.json]: CQ - https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20319

**Comment by evidolob** [comment]
CQ is approved, I'm going to merge this

**Pull request merged**